### PR TITLE
deps: update to mocha@10

### PR DESCRIPTION
**Pull request by barmac**
This migrates library to mocha@10 and exposes mjs and cjs bundles explicitly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,9 @@
   "name": "@bpmn-io/element-templates-validator",
   "version": "1.1.0",
   "description": "Validate element templates based on JSON Schema",
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
```
Comment (nikku) [inline package.json:7]: For modules the default extension `js` refers to a module. Any reason you still keep the `mjs` extension? Compatibility?

Comment (barmac) [reply package.json:7]: This might be a leftover from previous attempts to make it work. I will reverse the change, and let the CI test this.

Comment (barmac) [reply package.json:7]: BTW this PR was already merged via https://github.com/bpmn-io/element-templates-validator/pull/24

**Comment by barmac** [comment]
Merged via https://github.com/bpmn-io/element-templates-validator/pull/24

**Pull request closed without merging**